plottable.css

```diff
@@ -120,8 +120,45 @@ svg.plottable {
   fill: aliceblue;
   opacity: .7;
   stroke: #C0C8FF;
+}
+
+/* SelectionBox */
+.plottable .selection-box-layer .selection-area {
+  fill: black;
+  fill-opacity: 0.03;
+  stroke: #CCC;
+}
```

Comment:
This was what Design mentioned when I talked to them. I'll set up a second sync.